resolver.js

```diff
@@ -33,45 +34,72 @@ var domainForResolver = function domainForResolver(req) {
 }
 
 var defaultResolver = function resolver(host, url, req) {
+    var wikiDomain = domainForResolver(req)
+    if (wikiDomain === null) {
+        return Promise.resolve(null)
+    }
+
+    return fetchWikiForDomain(wikiDomain)
+        .then(function (response) {
```

Comment:
When testing locally making two repeated calls to the same domain results in the following stack trace (and then the container terminating):
```
/redbird/resolver.js:43
        .then(function (response) {
         ^

TypeError: fetchWikiForDomain(...).then is not a function
    at ReverseProxy.resolver (/redbird/resolver.js:43:10)
    at ReverseProxy.resolve (/redbird/node_modules/redbird/lib/proxy.js:541:41)
    at ReverseProxy._getTarget (/redbird/node_modules/redbird/lib/proxy.js:601:15)
    at Server.<anonymous> (/redbird/node_modules/redbird/lib/proxy.js:184:11)
    at Server.emit (node:events:369:20)
    at parserOnIncoming (node:_http_server:936:12)
    at HTTPParser.parserOnHeadersComplete (node:_http_common:129:17)

```

presumably this is because when doing a cache retrieval we return an actual response object rather than a promise that will eventually resolve to it? See line 72